### ISSUE TITLE
Can't add/update memo field (long text)

### ISSUE THREAD

**Issue by imidimi** [luads/php-xbase#91]
Reading memo for a record is working fine, but adding or updating it returns this error when i try to view it.

```
Warning:  fseek() expects parameter 2 to be integer, float given in <b>C:\xampp\htdocs\myPcmrp\lib\vendor\hisamu\php-xbase\src\XBase\Memo\FoxproMemo.php on line 38
Warning:  fread(): Length parameter must be greater than 0 in C:\xampp\htdocs\myPcmrp\lib\vendor\hisamu\php-xbase\src\XBase\Memo\FoxproMemo.php on line 42
```
I am attaching the table along with its FPT file. Field name is 'comment'. You can try updating or adding in this table.
[addrbook.zip](https://github.com/luads/php-xbase/files/5209325/addrbook.zip)

Here is the code i m using to add new row to table

```
$tableA = get($addressTable);
$allColumns = $tableA->getColumns();

foreach ($allColumns as $key => $column) {
    $allColumns[$key] = $column->getType();
}

$tableB = add($addressTable);
$tableB->openWrite();
$newRecord = $tableB->appendRecord();

foreach ($allColumns as $key => $column) {
    $newRecord->{$key} = isset($_POST[$key]) ? $_POST[$key] : '';
}

$tableB->writeRecord();
$tableB->pack();
$tableB->close();
unset($newRecord);
```

**Comment by gam6itko** [comment]
I'll fix it feature at the week

**Comment by ludvikjan** [comment]
I have the same problem. By append row to empty table, the first row stay empty and data are writen to next row.

**Comment by gam6itko** [comment]
I'm working on new approach to write data to table (create and update rows). I'll release it soon.

@ludvikjan I will appreciate if you give me the table file, and the code example to test and fix this issue.

**Comment by ludvikjan** [comment]
There is my example. After write data to empty table the header is not correct and in first row missing data.

`
copy('emptyTable.dbf', '/tmp/table.dbf');
copy('emptyTable.fpt', '/tmp/table.fpt');

$table = new WritableTable("/tmp/table.dbf", null, 'CP1250');
$table->openWrite();

$data = ['str1', 'str2', 'str3', 'str4'];

foreach($data as $value) {
    $record = $table->appendRecord();
    $record->zkratka = $value;
    $table->writeRecord();
}

$table->close();
`

[emptyTable.zip](https://github.com/luads/php-xbase/files/5274460/emptyTable.zip)

**Comment by gam6itko** [comment]
I think I fixed this bug. I ask you to check it works correctly. You need to use `dev-set_memo` version. I'm waitinf for your feedback

**Comment by ludvikjan** [comment]
@gam6itko in looks like it is OK.

**Comment by gam6itko** [comment]
fixed in #92
